freqtrade/exchange/exchange.py

```diff
@@ -2343,9 +2504,10 @@ def _get_trade_pagination_next_value(self, trades: List[Dict]):
             return trades[-1].get('timestamp')
 
     async def _async_get_trade_history_id(self, pair: str,
-                                          until: int,
+                                          until: Optional[int],
                                           since: Optional[int] = None,
-                                          from_id: Optional[str] = None) -> Tuple[str, List[List]]:
+                                          from_id: Optional[str] = None,
+                                          stop_on_from_id: Optional[bool] = True) -> Tuple[str, List[List]]:  # noqa
```

Comment:
`stop_on_from_id` is unused. This parameter is newly dragged through the whole call structure ... was there an actual use (which got lost or forgotten to implement), or is it a remainder that should be removed?